sale_promotion_rule/README.rst

```diff
@@ -6,10 +6,20 @@
 Sale Promotion Rule
 ===================
 
-This module add the concept of promotion rule that can be applied on the sale order.
+This module add the concept of promotion rules that can be applied on the sale
+order.
 
-for now only coupon with discount amount have been implemented.
+Two kinds of rules are implemented:
+ * automatic
+ * coupon
 
+Automatic rules are applied/recomputed automatically for a sale order when the
+user click on the button "Apply discount" into the view form. Depending of the
```

Comment:
s/user click on the button "Apply discount" into the view form/user clicks on the "Apply discount" button in the form view